src/state/_activeMetadata/constants.js

```diff
@@ -0,0 +1,43 @@
+import Select from '../Select';
+import ActionTypes from '../../constants/ActionTypes';
+
+export const STORES_TO_ENSURE_WITH_FILTER_BY_ACTIVE = {
```

Comment:
Each stor has specific structure?

Can you add some doc? 

first param: getSubstate, required
second param: storeId, required
thirth param: ActionTypes, required
next optional